<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopicWidget</class>
 <widget class="QWidget" name="TopicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageEdit">
      <layout class="QHBoxLayout" name="horizontalLayoutEdit">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSaveName">
         <property name="text">
          <string>Save topic name</string>
         </property>
         <property name="icon">
          <iconset resource="NotesManager.qrc">
           <normaloff>:/icons/fatcow/32x32-grey/file_save_as.png</normaloff>:/icons/fatcow/32x32-grey/file_save_as.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageLabel">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonEditName">
         <property name="text">
          <string>Change topic name</string>
         </property>
         <property name="icon">
          <iconset resource="NotesManager.qrc">
           <normaloff>:/icons/fatcow/32x32-grey/pencil.png</normaloff>:/icons/fatcow/32x32-grey/pencil.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButtonAddNote">
     <property name="text">
      <string>Neue Notiz hinzuf√ºgen</string>
     </property>
     <property name="icon">
      <iconset resource="NotesManager.qrc">
       <normaloff>:/icons/fatcow/32x32-grey/add.png</normaloff>:/icons/fatcow/32x32-grey/add.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="listViewNotes"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="NotesManager.qrc"/>
 </resources>
 <connections/>
</ui>
